modules/user/client-react/components/LoginForm.native.jsx

```diff
@@ -15,29 +15,30 @@ const loginFormSchema = {
   usernameOrEmail: [required, minLength(3)],
   password: [required, minLength(settings.auth.password.minLength)]
 };
-const { github, facebook, linkedin, google } = settings.auth.social;
+const { github, facebook, linkedin, google, googleExpo } = settings.auth.social;
 
 const renderSocialButtons = (buttonsLength, t) => {
   return buttonsLength > 2 ? (
     <View style={{ flexDirection: 'row', justifyContent: 'space-between', alignItems: 'center' }}>
       {facebook.enabled && <FacebookButton text={t('login.fbBtn')} type="icon" />}
-      {google.enabled && <GoogleButton text={t('login.googleBtn')} type="icon" />}
+      {(google.enabled || googleExpo.enabled) && <GoogleButton text={t('login.googleBtn')} type="icon" />}
```

Comment:
This solution is the cause of splitting `googleExpo` settings from the `google`.

Reply:
gone